desktop/components/commercial_filter/filters/keyword/index.jade

```diff
@@ -0,0 +1,13 @@
+#keyword-search-bar-container.typeahead
+  input#keyword-search-bar-input.bordered-input(
+    placeholder='Search…'
+    tabindex='1'
+    itemprop='query-input'
+    type='text'
+    name='keyword'
+    value= term
```

Comment:
I'm not entirely sure what i should be using as the value property here... this seems to work fine. I think i cargo culted this from the main search bar so maybe using `term` here will interact badly with that.